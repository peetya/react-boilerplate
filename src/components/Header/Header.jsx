import React from 'react';
import { Link, NavLink } from 'react-router-dom';

const Header = () => (
    <div className="app-header">
        <nav className="navbar navbar-expand-lg navbar-light bg-light">
            <div className="container">
                <Link className="navbar-brand" to="/">My App</Link>
                <div className="collapse navbar-collapse" id="navbarSupportedContent">
                    <ul className="navbar-nav mr-auto">
                        <li className="nav-item">
                            <NavLink className="nav-link" exact to="/">Home</NavLink>
                        </li>
                        <li className="nav-item">
                            <NavLink className="nav-link" exact to="/test">Test</NavLink>
                        </li>
                    </ul>
                </div>
            </div>
        </nav>
    </div>
);

export default Header;
