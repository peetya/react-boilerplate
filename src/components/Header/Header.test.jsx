import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router';
import Header from './Header';

test('matches snapshot', () => {
    const component = renderer.create(
        <MemoryRouter>
            <Header />
        </MemoryRouter>
    );
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
});
