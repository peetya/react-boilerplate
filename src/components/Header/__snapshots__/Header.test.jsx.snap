// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="app-header"
>
  <nav
    className="navbar navbar-expand-lg navbar-light bg-light"
  >
    <div
      className="container"
    >
      <a
        className="navbar-brand"
        href="/"
        onClick={[Function]}
      >
        My App
      </a>
      <div
        className="collapse navbar-collapse"
        id="navbarSupportedContent"
      >
        <ul
          className="navbar-nav mr-auto"
        >
          <li
            className="nav-item"
          >
            <a
              aria-current="page"
              className="nav-link active"
              href="/"
              onClick={[Function]}
              style={Object {}}
            >
              Home
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              aria-current={null}
              className="nav-link"
              href="/test"
              onClick={[Function]}
            >
              Test
            </a>
          </li>
        </ul>
      </div>
    </div>
  </nav>
</div>
`;
